src/app/(pages)/students/_components/Section.tsx

```diff
@@ -18,7 +18,7 @@ export default function Section({ team }: { team: TeamDataType }) {
       gap="less"
       titleSize="sm"
     >
-      <div className="grid grid-cols-2 gap-8 md:grid-cols-4 lg:grid-cols-6">
+      <div className="-mx-2 grid grid-cols-2 gap-2 md:grid-cols-4 md:gap-8 lg:grid-cols-6">
```

Comment:
```suggestion
      <div className="-mx-2 md:mx-auto grid grid-cols-2 gap-2 md:grid-cols-4 md:gap-8 lg:grid-cols-6">
```
we should add a breakpoint and send left and right margin back to normal if our screen size is not mobile